.github/workflows/rust.yml

```
@@ -18,12 +22,31 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: cargo build
-      run: (cd fiat-rust; cargo build --verbose)
+      run: cargo build --verbose
     - name: cargo test
-      run: (cd fiat-rust; cargo test --verbose)
-    - name: cargo test (no_std)
-      run: (cd fiat-rust; cargo test --no-default-features --verbose)
+      run: cargo test --verbose
     - name: curve25519-dalek test
-      run: etc/ci/test-fiat-rust-curve25519-dalek.sh
+      run: ../etc/ci/test-fiat-rust-curve25519-dalek.sh
     - name: crate-crypto/Ed448-Goldilocks test
-      run: etc/ci/test-fiat-rust-crate-crypto-Ed448-Goldilocks.sh
+      run: ../etc/ci/test-fiat-rust-crate-crypto-Ed448-Goldilocks.sh
+
+  build-nostd:
+    name: Build on no_std target (thumbv7em-none-eabi)
+    runs-on: ubuntu-latest
+
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

I think we want
```suggestion
      group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}-${{ github.job }}
```
on both this and the one above, or else the jobs are going to cancel each other

---

Fixed